src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -20,17 +20,17 @@
     <RestorePackages>true</RestorePackages>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;COMPILERCORE</DefineConstants>
+    <DefineConstants>$(DefineConstants);COMPILERCORE</DefineConstants>
```

Comment:
Wouldn't it make sense to pull this out of the configuration/platform-specific `PropertyGroup` like you did in CodeAnalysis.Desktop.csproj?